addons/web/static/src/legacy/scss/form_view.scss

```diff
@@ -474,7 +474,7 @@ $o-form-label-margin-right: 0px;
 
         @mixin dropdownButtonsFix {
             .btn.oe_stat_button.dropdown-item {
-                height: 44px !important;
+                height: 50px !important;
```

Comment:
Why did you do this? 🤔

Reply:
check first point on Dit review on Feb 9, she added the problem on stats box size